src/test/java/htsjdk/samtools/SamReaderFactoryTest.java

```diff
@@ -38,6 +39,9 @@
 public class SamReaderFactoryTest extends HtsjdkTest {
     private static final File TEST_DATA_DIR = new File("src/test/resources/htsjdk/samtools");
 
+    private static final String HTSGET_ENDPOINT = "htsget://127.0.0.1:3000/reads/";
```

Comment:
Lets hardcore these in 1 place only and reference them here instead.  Otherwise it's a pain if we change the location.